blockchain/binance-smart-chain.go

```diff
@@ -1,176 +0,0 @@
-package blockchain
```

Comment:
Deleted because the current ETH implementation would replace this now, so no need to maintain two exact identical implementations.

If there ever is a need to change the implementation for BSC, we can bring it back again.